.github/workflows/deploy-to-library.yml

```diff
@@ -0,0 +1,45 @@
+name: SYNC
+env:
+  # Username/email used to commit to the Core Components Library
+  PROJECT_DIR: core-components
+ 
+# Only run on a push to this branch
+on:
+  push:
+     branches: [ development ]
+ 
+jobs:
+  build:
+    runs-on: ubuntu-latest
+ 
+    steps:
+      # Checkout this project into a sub folder
+      - uses: actions/checkout@v2
+        with:
+          path: sub
+      # Cleanup sub project
+      - name: Clean project
+        run: |
+          git -C sub log --format="%an : %s" -n 1 > commit.txt
+          rm -rf sub/.git
+          rm -rf sub/.github
+      # Set global git configuration
+      - name: Set git config
+        run: |
+          git config --global credential.helper cache
+          git config --global user.email ${{ secrets.LIBRARY_USER_EMAIL }}
```

Comment:
As per https://docs.github.com/en/actions/reference/encrypted-secrets#using-encrypted-secrets-in-a-workflow, secrets should be assigned to environment variables.